src/toil_scripts/adam_pipeline/spark_toil_script.py

```diff
@@ -241,6 +272,7 @@ def stop(self):
         log.write("stop masters\n")
         log.flush()
 
+        call(["docker", "exec", self.sparkContainerID, "rm", "-r", "/ephemeral/spark"])
```

Comment:
I think it'd be preferable to stop the spark container, and to then shutil.rmtree("/mnt/ephemeral/spark"). I'm just imagining that something uncouth could happen if we yanked the directory out while Spark was still running.